### PR TITLE
created routing and navbar for basic navigation

### PR DESCRIPTION
**Pull request by FlowerAlex**

### CODE REVIEW

**Comment by FlowerAlex** [comment]
I changed these tests, but now it's not important, because we haven't yet any data on screen

File: .github/workflows/hotel.yml [hunk 1]
```diff
@@ -34,11 +34,11 @@ jobs:
 
       - name: Run the tests
         run: npm test
-        working-directory: ./client
+        working-directory: ./hotel
 
       - name: Build
         run: npm run build
-        working-directory: ./client
+        working-directory: ./hotel
```
Comment (areknoster) [inline .github/workflows/hotel.yml:41]: very good catch, thank you!

File: hotel/src/App.js [hunk 2]
```diff
@@ -1,10 +1,63 @@
...
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+  }
+}));
+
+export default function ButtonAppBar() {
```
Comment (areknoster) [inline hotel/src/App.js:30]: Its name should be `App`

**Review by areknoster — APPROVED** [review]
Very good job overall

File: hotel/package.json [hunk 3]
```diff
@@ -3,11 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
```
Comment (areknoster) [inline hotel/package.json:7]: I dont like it

File: hotel/public/index.html [hunk 4]
```diff
@@ -24,6 +23,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```
Comment (areknoster) [inline hotel/public/index.html:27]: very good

File: hotel/package.json (hunk 3 above)
Comment (mikevelko) [reply hotel/package.json:7]: @areknoster it's just a components library, for me it's fine

**Pull request merged**